src/js/item.ts

```diff
@@ -4,11 +4,13 @@ import * as azad_entity from './entity';
 import * as extraction from './extraction';
 import * as util from './util';
 import * as order_header from './order_header';
+import { IRequestScheduler, IResponse } from './request_scheduler';
```

Comment:
Are you thinking "ugh! he's used * everywhere"? Do you see it as a big problem? I've been pretty restrictive in what gets exported from each module so have (until I saw this line) blithely unconcerned about accepting all the names exported, particularly as they're guarded by the module alias. I like having the alias in front of the name, so you know where it's coming from.